docs/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error.md

```diff
@@ -748,7 +749,7 @@ We're sorry, but we don't have any specific information about this error. You ca
 Provides a QA website for questions about and issues with Visual C#.  
 
  Also see:  
- [Additional Resources for Visual C# Programmers](../getting-started/index.md)  
+ [Additional Resources for Visual C# Programmers](../tour-of-csharp/index.md)  
```

Comment:
nit: This section seems odd. Since you're editing the link, might as well as move it 3 lines down to the **See also** section